scripts/build_a3_package.py

```
@@ -0,0 +1,241 @@
+        return False
+    if relative_path.suffix.lower() in PROHIBITED_SUFFIXES:
+        return False
+    if relative_path.parts[:2] == ("data", "semeval2014") and relative_path.name != ".gitkeep":
+        return False
+    return True
+
+
+def tracked_source_entries(repo: Path) -> list[PackageEntry]:
+    entries: list[PackageEntry] = []
+    result = subprocess.run(
+        ("git", "ls-files", "-z"),
```

Ruff (0.16.0)

[error] Starting a process with a partial executable path

(S607)